ansible/roles/ooniprobe/tasks/install-git.yml

```diff
@@ -34,4 +13,4 @@
     name={{ item }}
   with_items:
     - cffi
-    - ooniprobe
+    - 'git+https://github.com/TheTorProject/ooni-probe.git#egg=ooniprobe'
```

Comment:
This is ooniprobe git version install task, for the stable ooniprobe there is another [task](https://github.com/TheTorProject/ooni-sysadmin/blob/master/ansible/roles/ooniprobe/tasks/install.yml)  which install ooniprobe package from Debian stretch repository.